src/Algolia.Search/Clients/AlgoliaConfig.cs

```diff
@@ -26,6 +26,7 @@
 using Algolia.Search.Transport;
 using System.Collections.Generic;
 using System.Reflection;
+using Algolia.Search.Utils;
```

Comment:
I don't get this new import. 🥺

Reply:
It's because I fixed the namespace of the internal class `Defaults` which is now in `Algolia.Search.Utils`.
This file/class is using it so it needed the new using :)